deploy/scripts/src/txma/requestGenerator/txmaReqFormat.ts

```diff
@@ -16,6 +16,24 @@ export interface AuthLogInSuccess {
   }
 }
 
+export interface AuthAuthorisationInitiated {
```

Comment:
This is not the correct schema for the `AUTH_AUTHORISATION_INITIATED` event. Below is the correct template:

```
export interface AuthAuthorisationInitiated {
  event_name: string
  client_id: string
  component_id: string
  timestamp: string
  event_timestamp_ms: string
  extensions: {
    'client-name': string
  }
  user: {
    govuk_signin_journey_id: string
    ip_address: string
    session_id: string
    persistent_session_id: string
  }
}
```